src/index.js

```diff
@@ -1,8 +1,8 @@
 'use strict'
 
-const { Pool } = require('pg')
-const escape = require('pg-escape')
-const { map, uniq, difference } = require('lodash')
+import {Pool} from 'pg'
```

Comment:
should this be 
```suggestion
import { Pool } from 'pg'
```
?

Reply:
neither standardjs or jshint corrected it, and I was hesitant to run WebStorms reformat. But I agree, it should have spaces.